package.json

```diff
@@ -250,7 +250,7 @@
     "@grafana/lezer-traceql": "0.0.6",
     "@grafana/monaco-logql": "^0.0.7",
     "@grafana/runtime": "workspace:*",
-    "@grafana/scenes": "^1.15.0",
+    "@grafana/scenes": "portal:/home/torkel/dev/scenes/packages/scenes",
```

Comment:
this is probably not meant to be added?

Reply:
@gabor it's waiting for a scenes PR to be merged, so will need to updated anyway after that